test/Microsoft.AspNetCore.Mvc.FunctionalTests/MvcSandboxTest.cs

```diff
@@ -17,29 +16,14 @@ public MvcSandboxTest(MvcSampleFixture<MvcSandbox.Startup> fixture)
 
         public HttpClient Client { get; }
 
-        [Theory]
-        [InlineData("")]                        // Shared/MyView.cshtml
-        [InlineData("/")]                       // Shared/MyView.cshtml
-        [InlineData("/Home/Index")]             // Shared/MyView.cshtml
```

Comment:
They test the exact same page. All we care about is to ensure that the Sandbox compiles and runs fine.